Missionframework/modules/16_garrison/fnc/fn_garrison_preInit.sqf

```diff
@@ -64,14 +64,30 @@ if (isServer) then {
                 if (!KPLIB_garrison_presetInitE) then {
                     KPLIB_garrison_presetInitE = true;
                 } else {
-                    [_setting] call KPLIB_fnc_garrison_reInitSectors;
+                    if (isServer) then {
+                        [] call KPLIB_fnc_init_loadPresets;
+                        [_setting] call KPLIB_fnc_garrison_reInitSectors;
+                    } else {
```

Comment:
This part could use some comments. Especially that you're calling other modules here.